letsencrypt/tests/renewer_test.py

```diff
@@ -0,0 +1,684 @@
+"""Tests for letsencrypt/renewer.py"""
+
+import configobj
+import datetime
+import mock
+import os
+import tempfile
+import pkg_resources
+import pytz
+import shutil
+import unittest
+
+ALL_FOUR = ("cert", "privkey", "chain", "fullchain")
```

Comment:
You probably want to use `storage.ALL_FOUR` throughout this file, instead of redefining.